packages/terra-form-checkbox/src/Checkbox.scss

```diff
@@ -7,7 +7,7 @@
     font-family: inherit;
     margin-bottom: var(--terra-form-checkbox-container-margin-bottom, 0);
     margin-left: var(--terra-form-checkbox-container-margin-left, 0);
-    margin-right: var(--terra-form-checkbox-container-margin-right, 2rem);
+    margin-right: var(--terra-form-checkbox-container-margin-right, 0);
```

Comment:
Removing this will result in inline checkboxes having no padding / margins between them by default. It would be worth discussing with UX how this should be handled. Whether that is by the user, by applying margins only to inline checkboxes, by a CheckboxGroup component, or some other way. 

![image](https://user-images.githubusercontent.com/6720991/36106346-b5295924-0fdc-11e8-8705-161d7749ab4b.png)